### DISCUSSION TITLE
Trouble formatting lists

### DISCUSSION THREAD

**Discussion by epayum200** [squidfunk/mkdocs-material#6048] [Q&A]
I have some markdown like this: 
```
# FOO

1. one
2. two
    1. eye
       1. ay
       2. bee
       3. sea
```

but it gets rendered as:
```
FOO

1. one
2. two 1. eye 1. ay 2. bee 3. sea
```

Is the only workaround to insert newlines between each item?

**Comment by alexvoss** [comment]
I am afraid I cannot reproduce this in a project I have. Can you provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) for this, please, or at least send the original markdown file and your `mkdocs.yml`?

**Comment by facelessuser** [comment]
Each level needs to be indented 4 spaces. It appears your last level is only indented 3 spaces.

**Reply by alexvoss** [reply]
Well spotted. Yet, this does not seem to cause a problem when I try to replicate. Possible that something else is going on?

**Reply by facelessuser** [reply]
I'm not sure what you are doing, but if you don't indent _at least_ 4 spaces, you will lose levels.

I also don't know if what they posted is exactly what they have in their own docs as I was able to get an issue with losing a list level, but I didn't get the exact issue they described.

Based on what they've provided, that is the only issue that I see. If there is more at play, they will need to provide a complete, minimal reproduction.

**Reply by alexvoss** [reply]
You are right, the list structure with 3 spaces comes out as:

```
1. one
2. two
      1. eye
      2. ay
      3. bee
      4. sea
```
Sorry,  was comparing what I got to what the OP got, not to the input. That is why I did not spot the issue with the number of spaces. Am somewhat surprised, though, that Markdown is so picky. Hey ho.

**Reply by facelessuser** [reply]
Depends on the parser. Python Markdown is an old school Markdown parser compared to the recent CommonMark implementations. Python Markdown took the original spec and followed the general 4 space indentation rule for blocks. Over time, some people created some implementations that would allow 2+ spaces for lists. CommonMark came along and people argued about what exactly should be done (based on the majority of opinions by those creating the new spec). 

For instance, GitHub uses a CommonMark parser, so it allows 2+ spaces for lists, but you must have 4+ spaces for code blocks. I find this inconsistency odd, but it is what is. Some people really prefer lists to use 2+ spaces 🤷🏻.

Anyways, as far as Python Markdown is concerned 4+ spaces is needed for a new list level by default.
